pkg/gce-cloud-provider/compute/gce-compute.go

```diff
@@ -38,9 +39,12 @@ const (
 	waitForImageCreationTimeOut    = 5 * time.Minute
 	diskKind                       = "compute#disk"
 	cryptoKeyVerDelimiter          = "/cryptoKeyVersions"
+	// Example message: "[pd-standard] features are not compatible for creating instance"
+	pdDiskTypeUnsupportedPattern = `\[([a-z-]+)\] features are not compatible for creating instance`
```

Comment:
Do we also want to surface the wrong VM type?

Reply:
Yes, that's a good idea. Updated this to print out the machine type, rather than just the machine family.